components/browser/session/src/main/java/mozilla/components/browser/session/storage/DefaultSessionStorage.kt

```diff
@@ -68,9 +68,10 @@ class DefaultSessionStorage(
     }
 
     @Synchronized
-    override fun restore(engine: Engine, sessionManager: SessionManager): Boolean {
+    override fun restore(sessionManager: SessionManager): Boolean {
         return try {
-            getFile().openRead().use {
+            val engine = sessionManager.engine
+            getFile(engine.name()).openRead().use {
```

Comment:
why the assignment to engine here first? for readability?

Reply:
yes, readability as I need it in inside the block as well, but probably not worth it. I will update. I n need to update this anyway with some test coverage for our engine type (we don't have any tests there yet.)